docs/xls-30d-amm/transaction-types/ammcreate.md

```diff
@@ -31,7 +31,7 @@ Creates both an [AMM object](../ledger-object-types/amm.md) and a [special Accou
     },
     "Amount2" : "250000000",
     "Fee" : "10",
-    "Flags" : 2147483648,
+    "Flags" : 0,
```

Comment:
This is [`tfFullyCanonicalSig`](https://xrpl.org/transaction-common-fields.html#global-flags) which is a (the only) global flag, and still allowed on all transaction types, if irrelevant nowadays.

It _might_ be better to add it than not just in case you're using a dev network that doesn't have the RequireFullyCanonicalSig amendment enabled for some reason?